.github/workflows/test-component-library-vizro-core.yml

```
@@ -0,0 +1,66 @@
+name: Integration tests Component Library
+
+defaults:
+  run:
+    working-directory: vizro-core
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches:
+      - main
+
+env:
+  PYTHONUNBUFFERED: 1
+  FORCE_COLOR: 1
```

```suggestion
  FORCE_COLOR: 1
  PYTHON_VERSION: "3.12"
```

Just to match how we do it elsewhere. This makes it easier to bump Python version everywhere.